homeassistant/components/homematicip_cloud/sensor.py

```diff
@@ -8,6 +8,7 @@
 from homematicip.aio.device import (
     AsyncBrandSwitchMeasuring,
     AsyncEnergySensorsInterface,
+    AsyncFloorTerminalBlock12,
```

Comment:
HI @barryvdh 
can you please add imports AsyncFloorTerminalBlock6, AsyncFloorTerminalBlock10 and AsyncWiredFloorTerminalBlock12 to imports. All are nearly the same but with different amount of channels.

Reply:
This is done in https://github.com/home-assistant/core/pull/122759/commits/03b46cd5a6cc0f406f3f09919b5947c34ad10cd1